rust-runtime/aws-smithy-http-server/src/protocol/mod.rs

```diff
@@ -257,6 +257,13 @@ mod tests {
         assert!(matches!(result.unwrap_err(), MissingContentTypeReason::ToStrError(_)));
     }
 
+    #[test]
+    fn valid_params_content_type() {
```

Comment:
nit:

```suggestion
    fn valid_content_type_header_classifier_params() {
```

just to match the naming pattern of the other tests that also contain the name of the function being tested

Reply:
I matched it to the ones above that were doing something_something_content_type, but happy to rename.